src/common.js

```diff
@@ -245,19 +245,21 @@ function randomFileName() {
 exports.randomFileName = randomFileName;
 
 // objectAssign(target_obj, source_obj1 [, source_obj2 ...])
-// Ponyfill for Object.assign
+// Polyfill for Object.assign
```

Comment:
Wow, I had no idea that was a thing. Let me change it back but put quotes around it.